### PR TITLE
Gravity fix

### PR DESCRIPTION
**Pull request by EmilMadsen90**
- moved gravity instantiation from the `compile-rbd` to the `calibrate` command.
- restructured duplicate code from classes `RigidBodyDynamics` (rigid_body_dynamics.py) and `JointDynamics` (joint_dynamics.py) in an abstract base class `LinearSystem` (linear_system.py)

### CODE REVIEW

**Comment by EmilMadsen90** [comment]
Failed due to a violation of the execution time limit (360 minutes) of Github. I will look into the efficiency of the code to see if something changed drastically compared to prior releases. Thus, do not continue with this pull request before any updates from my side.

**Comment by clagms** [comment]
These time limits makes it almost useless to use github actions as way to test the long running code. If our code ran fast we could easily run it on our machines...

**Comment by clagms** [comment]
This afternoon I'll run the tests on my machine Emil! Will let you know how it goes. If it passes, then we can merge.

**Comment by EmilMadsen90** [comment]
> These time limits makes it almost useless to use github actions as way to test the long running code. If our code ran fast we could easily run it on our machines...

To elaborate on the commit message, the reason for the slow code was a bummer from my side, where I had disabled multiprocessing while debugging. It is enabled again now. I don't think it makes sense to make process changes such as discarding Github actions based on this incident. Rather, we could make tests measuring the efficiency of the AURT commands and compare running times to prior commits?

**Comment by EmilMadsen90** [comment]
Tests passed on all platforms, so @clagms and @Daniella1 you can continue with the pull request.

File: aurt/joint_dynamics.py [hunk 1]
```diff
@@ -70,9 +78,10 @@ def _coulomb_friction_basis(self):
         if self.load_model.lower() == 'none':
             return [[h[j]] for j in range(self.n_joints + 1)]
         elif self.load_model.lower() == 'square':
-            return [[h[j], h[j] * self._tauJ[j] ** 2] for j in range(self.n_joints + 1)]
+            return [[h[j], h[j]*self._tauJ[j]**2] for j in range(self.n_joints + 1)]
+            # return [[*h[j], *(sp.Matrix(h[j])*self._tauJ[j]**2).tolist()] for j in range(self.n_joints + 1)]
```
Comment (clagms) [inline aurt/joint_dynamics.py:82]: Maybe remove this commented out code.

File: aurt/paper_code_and_plots.py [hunk 2]
```diff
@@ -0,0 +1,309 @@
+
```
Comment (clagms) [inline aurt/paper_code_and_plots.py:1]: Is this file being run as part of the CI? If not, it would be good to make it part of it (with plotting disabled)

**Review by clagms — APPROVED** [review]
It looks good. Just clean up the commented out code a bit. We have that in several places and it does not look good. Usually it's the result of work in progress and has in the meantime become irrelevant...

File: aurt/paper_code_and_plots.py (hunk 2 above)
Comment (EmilMadsen90) [reply aurt/paper_code_and_plots.py:1]: I think this file was mistakenly added to the repo. We currently use the plot functions defined in robot_calibration. @Daniella1 what do you think?

Comment (Daniella1) [reply aurt/paper_code_and_plots.py:1]: I don't think the file was supposed to be pushed to the repo.

**Comment by EmilMadsen90** [comment]
changed gravity instantiation from rigid_body_dynamics to robot_calibration

**Pull request merged**